packages/auth/src/types.ts

```diff
@@ -1,4 +1,6 @@
-﻿import { CognitoUser } from "amazon-cognito-identity-js";
+﻿// @TODO Move this back to aws-amplify-hooks repo
```

Comment:
this todo is still here is it still relevant?

Reply:
Oh I forgot to remove it. thanks.